### PR TITLE
Depth metal

### PR DESCRIPTION
**Pull request by omni-viral**
:skull:  :metal:

### CODE REVIEW

File: src/backend/metal/src/command.rs [hunk 1]
```diff
@@ -137,6 +138,9 @@ impl CommandBufferInner {
         if let Some(ref pipeline_state) = self.pipeline_state {
             encoder.set_render_pipeline_state(pipeline_state);
         }
+        if let Some(ref depth_stencil_state) = self.depth_stencil_state {
```
Comment (kvark) [inline src/backend/metal/src/command.rs]: we also need to set the state if the encoder is already active in `bind_graphics_pipeline`

**Comment by kvark** [comment]
@omni-viral ping?

**Comment by omni-viral** [comment]
@kvark got a bit busy. I'll fix this soon.

File: src/backend/metal/src/command.rs (hunk 1 above)
Comment (omni-viral) [reply src/backend/metal/src/command.rs]: Done

**Pull request merged**